src/components/blogs/Blogs.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const Blogs = () => {
+  return (
+    <div>
+      <Link to="/singleBolgPage">Lorem ipsem till Gunsu add her code</Link>
```

Comment:
typo in blog

Reply:
I add it as a temporary text till Gunsu push Blogs page because I need something I can hit on to direct me to the single blog page.
There will be no need for this when Gunsu push her files

Reply:
the route has a typo in it i think?